core/src/main/java/org/elasticsearch/index/translog/TranslogWriter.java

```diff
@@ -191,6 +205,14 @@ public int totalOperations() {
         return operationCounter;
     }
 
+    public long getMinSeqNo() {
```

Comment:
Yeah, I'd prefer not to expose these unless we're sure we need them because of the concurrency issues. Or just expose them for testing if we need them there.